iroha_logger/src/lib.rs

```diff
@@ -53,64 +53,72 @@ pub fn init(configuration: config::LoggerConfiguration) -> Option<Receiver<Telem
     }
 }
 
+/// Macro for sending telemetry info
+#[macro_export]
+macro_rules! telemetry_target {
+    () => {
+        concat!("telemetry::", module_path!())
```

Comment:
It is better to also do it like that here `concat!(iroha_logger::telemetry::TELEMETRY_TARGET_PREFIX, module_path!())`

Reply:
It won't work in such way, it is not literal. That is why it is changed.

Reply:
Don't know, though why it worked before, but compiler clearly complains about that

Reply:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=57ec380465640be22f41d9e24b235601

Reply:
It will add an allocation for each of the telemetry call. It clearly can be just a compilation time info.

Point is in using concat!. It requires for arguments to be string literals, that is why it can't be a constant, that is why i abstracted it to the separate macro like a constant.